src/app/legacy/psammead/psammead-image/src/index.jsx

```diff
@@ -63,9 +63,9 @@ export const Img = props => {
             />
           )}
           <StyledImg src={src} {...otherProps} />
-        </StyledPicture>
+        </StyledPicture>,
       )}
-      {pageType !== FRONT_PAGE && (
+      {pageType !== FRONT_PAGE && pageType !== HOME_PAGE && (
```

Comment:
```suggestion
      {![FRONT_PAGE, HOME_PAGE].includes(pageType) && (
```

Reply:
Originally tried this and typescript had a tantrum. Will try again as your syntax may be slightly different, let's see!

Reply:
Let's try and work through the TS tantrums as best we can 😸